iidm/iidm-api/src/main/java/com/powsybl/iidm/network/IdentifiableAdder.java

```diff
@@ -11,7 +11,7 @@
  *
  * @author Geoffroy Jamgotchian {@literal <geoffroy.jamgotchian at rte-france.com>}
  */
-public interface IdentifiableAdder<T extends Identifiable<T>, A extends IdentifiableAdder> {
+public interface IdentifiableAdder<T extends Identifiable<T>, A extends IdentifiableAdder> extends PropertiesHolder {
```

Comment:
Why is it needed for the `Identifiable` adder? Properties were already handled for `Identifiable`s.